cmd/kubectl-gadget/trace/signal.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 type SignalParser struct {
```

Comment:
Do you mean "Why not to use directly `utils.BaseParser` and do not create `SignalParser`?

If that's what you mean, I didn't use directly  `utils.BaseParser` because for Gadget with Flags, it will be extended with the `<Gadget>Flags`, example [here](https://github.com/kinvolk/inspektor-gadget/blob/c3db477afbf71be2c480e40152cceda3d354dfb6/cmd/kubectl-gadget/profile/cpu.go#L28-L37). So, I created it for all the gadgets for consistency in the code and let everything in place if in the future we need to add flags to a gadget.